Python/bhroBahroYeeshaCave.py

```diff
@@ -542,7 +542,7 @@ def DisablePole(self, age, fforward = 0):
         self.ageDict[age]['PoleCollider'].value.physics.suppress(1)
         if not fforward:
             vault = ptVault()
-            if type(vault) != type(None): #is the Vault online?
+            if vault is not None: #is the Vault online?
```

Comment:
This is safe to remove because `ptVault()` will never return `None` (maybe it did at some point 17+ years ago, however).